docs/architecture/providers/parameterized.md

```diff
@@ -1,14 +1,46 @@
 (parameterized-providers)=
 # Parameterized providers
 
-*Parameterized providers* are a feature of Pulumi that allows a caller to change
-a provider's behaviour at runtime in response to a
-[](pulumirpc.ResourceProvider.Parameterize) call. Where a
-[](pulumirpc.ResourceProvider.Configure) call allows a caller to influence
-provider behaviour at a high level (e.g. by specifying the region in which an
-AWS provider should operate), a [](pulumirpc.ResourceProvider.Parameterize) call
-may change the set of resources and functions that a provider offers (that is,
-its schema). A couple of examples of where this is useful are:
+*Parameterized providers* are a provider feature where a user can generate and
+use a custom SDK based on some custom input.
+
+Key rules:
+
+* Each provider parameterization is run in its own provider instance.
```

Comment:
This is true right now, for replacement parameterization, but I don't think (?) it will be for extensions. There, you might parameterize an instance multiple times. Not to say we shouldn't talk about this of course, just that we could contextualise to replacement.